relay-server/src/actors/processor.rs

```diff
@@ -2092,10 +2092,10 @@ impl EnvelopeProcessorService {
                         has_profile: state.has_profile,
                     };
 
-                    let mut extracted = extractor.extract(event)?;
+                    let mut extracted: ExtractedMetrics = extractor.extract(event)?;
 
                     // TODO: Move conditional tagging to generic metrics extraction
-                    let tagging_config = &state.project_state.config.metric_conditional_tagging;
+                    let tagging_config = state.project_state.metric_conditional_tagging();
```

Comment:
this is where we call our custom method that merges the global and project values. I wish we could make it impossible to get the project values directly, but IIRC, mixed visibility of fields in a struct is against our guidelines

Reply:
I would make the struct field private nonetheless.